editor/src/components/editor/actions/actions.spec.tsx

```diff
@@ -408,6 +408,8 @@ describe('UPDATE_FILE_PATH', () => {
     })
     expect(filesAndTheirImports).toMatchInlineSnapshot(`
       Object {
+        "/package.json": Array [],
```

Comment:
I'll have a little look into why these have been included all of a sudden, but they don't appear to be causing any harm.

Reply:
These appear to be being added because the parser is getting a little further along rather than just bailing out immediately. As a result something like `package.json` has a parse success, but everything is effectively empty in that result.